docs/source/science/NISAR/OPERA Sentinel-1 .ipynb

```diff
@@ -0,0 +1,661 @@
+{
```

Comment:
<p>What's the difference between the S2B, S2A, and L8 granules?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Are these references to Sentinel 2A/B and Landsat8? Can you point me to some documentation, this is a little confusing because those sensors are optical but this notebook is about radar data (Sentinel 1)

Reply:
<p>this is the wrong collection, you have the correct one below OPERA_L3_DISP-S1_V1</p>